packages/dbf/dbf.0.2.0/opam

```diff
@@ -0,0 +1,38 @@
+opam-version: "2.0"
+synopsis: "DBF format parsing"
+description:
+  "This is a small library meant to parse dBase database files. Many cases/variants are not handled, but it might already be useful."
+maintainer: ["philippe.veber@gmail.com"]
+authors: ["Léo Andrès" "Philippe Veber"]
+license: "ISC"
```

Comment:
There seems to be no license info in the repository. Can you confirm `ISC` is this is the license you are intending? Is it possible to clarify this on https://github.com/pveber/dbf?

Reply:
Done. I had forgotten to push the commit on the repo.

Reply:
Thanks!